jwt-auth/3.0/README.adoc

```diff
@@ -0,0 +1,36 @@
+---
+title: "MicroProfile JWT Auth 3.0"
```

Comment:
```suggestion
title: "MicroProfile JWT Auth 2.1"
```

Reply:
If there is no backward incompatible changes, it should be JWT Auth 2.2.

Reply:
@Emily-Jiang RSA keys 1024 size won't be accepted by default, it is not an API level change, but MP JWT 2.1 users will have to enable a property if they want to continue using the 1024 bit keys.
Also, we have removed the optional spec and TCK parts - that warrants 3.0 anyway, I believe this is what you also suggested in the MP JWT 7.0 task issue for MP-JWT.

`The release won't contain any backwards incompatible changes.` has to be dropped and changed to a more appropriate message, can you please recommend something ? thanks